apisix/upstream.lua

```diff
@@ -179,7 +179,12 @@ function _M.set_by_route(route, api_ctx)
         if not dis then
             return 500, "discovery " .. up_conf.discovery_type .. " is uninitialized"
         end
-        local new_nodes = dis.nodes(up_conf.service_name)
+
+        local new_nodes, err = dis.nodes(up_conf.service_name)
+        if not new_nodes then
+            return http_code_upstream_unavailable, "no valid upstream node: " .. (err or "nil")
```

Comment:
`http_code_upstream_unavailable`:

For static variables, use capital letters

Reply:
@membphis 
I have submitted another PR to use the number literal directly: https://github.com/apache/apisix/pull/3672, which is more consistent in the style.

Reply:
I changed my idea and followed your suggestion.